runner.go

```diff
@@ -97,8 +96,8 @@ func (rn *runner) run(requeueWorkers bool) {
 
 	select {
 	case <-rn.stopped.Done():
```

Comment:
I personally like the "return" here. Although it is not needed, it makes the intentions more clear.